modular_skyrat/modules/encounters/code/enclave_pirates.dm

```diff
@@ -1,86 +1,88 @@
 /datum/outfit/pirate/enclave_officer
 	name = "Imperial Enclave Officer"
 
-	head = /obj/item/clothing/head/soft/enclaveo
+	head = /obj/item/clothing/head/beret/sec/nri
 	glasses = /obj/item/clothing/glasses/sunglasses
 	ears = /obj/item/radio/headset/guild/command
+	mask = /obj/item/clothing/mask/gas/hecu2
 
-	uniform = /obj/item/clothing/under/enclaveo
-	suit = /obj/item/clothing/suit/armor/vest
-	suit_store = /obj/item/gun/ballistic/automatic/pistol/automag
+	uniform = /obj/item/clothing/under/costume/nri/captain
+	suit = /obj/item/clothing/suit/space/hev_suit/nri/captain
+	suit_store = /obj/item/gun/ballistic/automatic/ak25
 
 	gloves = /obj/item/clothing/gloves/combat
 
 	shoes = /obj/item/clothing/shoes/combat
 
-	belt = /obj/item/storage/belt/military/assault
-	back = /obj/item/storage/backpack/satchel/leather
-	backpack_contents = list(/obj/item/ammo_box/magazine/automag = 4)
+	belt = /obj/item/storage/belt/military/nri/captain/pirate_officer
+	back = /obj/item/storage/backpack/duffelbag/syndie/nri/captain
+	backpack_contents = list(/obj/item/storage/box/nri_survival_pack = 1, /obj/item/ammo_box/magazine/automag = 2, /obj/item/clothing/head/helmet/space/hev_suit/nri/captain = 1, /obj/item/gun/ballistic/automatic/pistol/automag = 1, /obj/item/crucifix = 1)
 
 	id = /obj/item/card/id/advanced
 	id_trim = /datum/id_trim/enclave/officer
 
 /datum/outfit/pirate/enclave_officer/post_equip(mob/living/carbon/human/equipped_human, visualsOnly)
 	. = ..()
-	equipped_human.hairstyle = "Crewcut"
-	equipped_human.hair_color = COLOR_ALMOST_BLACK
-	equipped_human.facial_hairstyle = "Shaved"
-	equipped_human.facial_hair_color = COLOR_ALMOST_BLACK
-	equipped_human.update_hair()
 
 /datum/id_trim/enclave/officer
-	assignment = "Imperial Enclave Officer"
+	assignment = "NRI Field Officer"
 
 /datum/outfit/pirate/enclave_trooper
 	name = "Imperial Enclave Trooper"
 
-	head = /obj/item/clothing/head/soft/enclave
+	head = /obj/item/clothing/head/helmet/rus_helmet
 	glasses = /obj/item/clothing/glasses/sunglasses
 	ears = /obj/item/radio/headset/guild
+	mask = /obj/item/clothing/mask/gas/hecu2
 
-	uniform = /obj/item/clothing/under/enclave
-	suit = /obj/item/clothing/suit/armor/vest/alt
-	suit_store = /obj/item/gun/ballistic/automatic/m16
+	uniform = /obj/item/clothing/under/costume/nri
+	suit = /obj/item/clothing/suit/armor/vest/russian
+	suit_store = /obj/item/gun/ballistic/automatic/akm
```

Comment:
NRI footsoldiers don't get ammo for the AK, so they're stuck with just the magazine from what i understand?

Reply:
I thought I've fixed this fuck.

Reply:
They do, actually - their belt subtype has all the needed ammo.